### DISCUSSION TITLE
index.htmlファイルがないindex.jsを読み込ませたいときは、どうしたらいいですか？

### DISCUSSION THREAD

**Discussion by lumijoe** [community/community#70299] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

index.htmlファイルがないindex.jsを読み込ませたいときは、どうしたらいいですか？

**Comment by queenofcorgis** [comment]
Hi @lumijoe , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Reply by lumijoe** [reply]
OK!I understand. Thank you for your response. Is it possible to use the index.js file as the entry point for GitHub Pages, or is it better to use GitHub for code management and Vercel for displaying the site, especially since index.js would make the site dynamic?I hope I'm explaining it well in English, I would be happy if it gets across.
